teuthology/test/integration/setup-openstack.sh

```diff
@@ -0,0 +1,572 @@
+#!/bin/bash
+
+function create_config() {
+    local network=$1
+    local subnet=$2
+    local private_key=$3
+    local nameserver=$4
+    local labdomain=$5
+    local ip=$6
+
+    local where=$(dirname $0)
+
+    cat > ~/.teuthology.yaml <<EOF
```

Comment:
I would happily do otherwise if there was a --config option but I don't think there is. In any case this is run on a machine that is created for the sole purpose of running the cluster. You're not supposed to run setup-openstack yourself. That would mess a lot more than just the ~/.teuthology.yaml ;-)